astro/src/content/blog/fusionauth-web-summit.mdx

```diff
@@ -0,0 +1,392 @@
+
+![Enter image alt description](/img/blogs/fusionauth-web-summit/night-summit-wide.png)
+
+### **Tips for Social Events**
+
+Social events are an excellent opportunity to network in a more relaxed setting.
+
+Wear your Web Summit badge all the time. I once connected with some people that saw I was from SendGrid whilst sitting by myself at a restaurant. I ended up joining their table and met and amazing group of people.
+
+**Don't Drink to Much**
+
+You won’t be at your best and it will make the next day very long, trust me ;)
+
+**Don’t Just Sell**
+
+Ask people what they are working on. Trust me great results come from listening to people.
+
+**Stay Humble and Thankful**
+
+E.g. “It was great meeting you, and good luck with your venture.”
+
+While it’s tempting to attend every social event, ensure you’re not compromising your productivity. Strike a balance between work and leisure to get the most out of the experience.
+
+## **Using Social Media Effectively**
+
+Share your experiences on social media using event hashtags and official Web Summit accounts. Engage with other attendees online (on socials and in the app) to expand your network. Tag people with accounts and hashtags or any people featured in your posts.
+
+### **Official Web Summit Accounts to Tag**
+
+X.com - [https://x.com/WebSummit](https://x.com/WebSummit)
+
+Instagram - [https://www.instagram.com/websummit/](https://www.instagram.com/websummit/)
+
+LinkedIn - [https://www.linkedin.com/company/web-summit/](https://www.linkedin.com/company/web-summit/)
+
+TikTok - [https://www.tiktok.com/@websummit](https://www.tiktok.com/@websummit)
+
+YouTube - [https://www.youtube.com/@websummit](https://www.youtube.com/@websummit)
+
+### **Hashtags**
+
+Hashtags to be used are as follows:
+
+#websummit #websummit2024 #websummitlisbon
+
+**Use Shorts**
+
+Instagram Reels, TikTok, YouTube Shorts and LinkedIn Shorts (if they are available to you) are well worth posting. You’re a great chance to be shared by the official Web Summit account, and who knows what investor, partner or customer might see it.
+
+***“Document. Don’t create”*** - Gary Vaynerchuck
+
+Here’s a [great article from Gary Vee](https://garyvaynerchuk.com/creating-content-that-builds-your-personal-brand/) about approaching social media.
+
+Even if you don’t post regularly, you can post a reel after the event documenting your experience. Here’s [one](https://www.youtube.com/watch?v=k2UGhmxvcbU) I did a few years ago.
+
+### **Video Interviews**
+
+Come across a great customer or partner, take a few minutes to interview them. You will need a good mic as it’s noisy. Whilst your phone’s camera will be good enough, the mic will not and would leave the footage unusable. Something simple like this mic on Amazon would be fine.
```

Comment:
```suggestion
If you come across a great customer or partner, take a few minutes to interview them. You will need a good mic as it’s noisy. Whilst your phone’s camera will be good enough, the mic will not and would leave the footage unusable. Something simple like this mic on Amazon would be fine.
```

Reply:
Looks like he forgot the link, I'd probably just remove the last sentence unless you have it lying around in a google doc.